.github/workflows/canary-release.yml

```
@@ -0,0 +1,31 @@
+name: "Canary release"
+
+on:
+  workflow_dispatch:
+    inputs:
+      label:
+        description: "Pick a label for the release, to be included in the version number — e.g. '1.0.0-\\${label}.x' (note: can't contain slashes!)"
```

```suggestion
        description: "Pick a label for the release, to be included in the version number"
```
The example confused more than helped, I think if you are able to get a comment with the version, we won't need it.